packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -53,6 +57,12 @@ export class Copy {
         ImperativeExpect.toBeDefinedAndNonBlank(options["from-dataset"].dsn, "fromDataSetName");
         ImperativeExpect.toBeDefinedAndNonBlank(toDataSetName, "toDataSetName");
 
+        const sourceIsPds = await this.isPDS(session, options["from-dataset"].dsn);
+        const targetIsPds = await this.isPDS(session, toDataSetName);
+        if (sourceIsPds && targetIsPds) {
+            return await this.copyPDS(session, options["from-dataset"].dsn, toDataSetName);
```

Comment:
I know we are focused on the APIs first, but I'm curious if we should update the corresponding help text on the CLI commands.

```shell

 EXAMPLES
 --------

   - Copy the data set named 'USER.FROM.SET' to the data set named 'USER.TO.SET':

      $ zowe zos-files copy data-set "USER.FROM.SET" "USER.TO.SET"

   - Copy the data set member named 'USER.FROM.SET(MEM1)' to the data set member named
   'USER.TO.SET(MEM2)':

      $ zowe zos-files copy data-set "USER.FROM.SET(mem1)" "USER.TO.SET(mem2)"

   - Copy the data set named 'USER.FROM.SET' to the data set member named 'USER.TO.SET(MEM2)':

      $ zowe zos-files copy data-set "USER.FROM.SET" "USER.TO.SET(mem2)"

   - Copy the data set member named 'USER.FROM.SET(MEM1)' to the data set named 'USER.TO.SET':

      $ zowe zos-files copy data-set "USER.FROM.SET(mem1)" "USER.TO.SET"

   - Copy the data set named 'USER.FROM.SET' to the data set named 'USER.TO.SET' and replace
   like-named members:

      $ zowe zos-files copy data-set "USER.FROM.SET" "USER.TO.SET" --replace

```